templates/plugin/ansible.env.erb

```diff
@@ -0,0 +1,13 @@
+ANSIBLE_CALLBACK_WHITELIST="<%= @callback %>"
+ANSIBLE_LOCAL_TEMP="<%= @working_dir %>"
+ANSIBLE_HOST_KEY_CHECKING="<%= @host_key_checking ? 'True' : 'False' %>"
+ANSIBLE_STDOUT_CALLBACK="<%= @stdout_callback %>"
```

Comment:
Why were we setting this? If I leave it in, I'm getting
```
SMART_PROXY_ANSIBLE_ENVIRONMENT_FILE=/etc/foreman-proxy/ansible.env /usr/libexec/foreman-proxy/ansible-runner-environment ansible-runner run --cmdline=-vvvvv -p playbook.yml .
ansible-playbook [core 2.12.2]
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/root/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /usr/lib/python3.8/site-packages/ansible
  ansible collection location = /root/.ansible/collections:/usr/share/ansible/collections
  executable location = /usr/bin/ansible-playbook
  python version = 3.8.12 (default, May 10 2022, 23:46:40) [GCC 8.5.0 20210514 (Red Hat 8.5.0-10)]
  jinja version = 2.10.3
  libyaml = True
Using /etc/ansible/ansible.cfg as config file
[DEPRECATION WARNING]: ANSIBLE_CALLBACK_WHITELIST option, normalizing names to 
new standard, use ANSIBLE_CALLBACKS_ENABLED instead. This feature will be 
removed from ansible-core in version 2.15. Deprecation warnings can be disabled
 by setting deprecation_warnings=False in ansible.cfg.
setting up inventory plugins
host_list declined parsing /tmp/d20221007-85919-17h90p3/inventory/hosts as it did not pass its verify_file() method
Parsed /tmp/d20221007-85919-17h90p3/inventory/hosts inventory source with script plugin
redirecting (type: callback) ansible.builtin.yaml to community.general.yaml
ERROR! Unexpected Exception, this is probably a bug: type 'NoneType' is not an acceptable base type
the full traceback was:

Traceback (most recent call last):
  File "/usr/bin/ansible-playbook", line 128, in <module>
    exit_code = cli.run()
  File "/usr/lib/python3.8/site-packages/ansible/cli/playbook.py", line 137, in run
    results = pbex.run()
  File "/usr/lib/python3.8/site-packages/ansible/executor/playbook_executor.py", line 119, in run
    self._tqm.load_callbacks()
  File "/usr/lib/python3.8/site-packages/ansible/executor/task_queue_manager.py", line 173, in load_callbacks
    self._stdout_callback = callback_loader.get(self._stdout_callback)
  File "/usr/lib/python3.8/site-packages/ansible/plugins/loader.py", line 807, in get
    return self.get_with_context(name, *args, **kwargs).object
  File "/usr/lib/python3.8/site-packages/ansible/plugins/loader.py", line 827, in get_with_context
    self._module_cache[path] = self._load_module_source(name, path)
  File "/usr/lib/python3.8/site-packages/ansible/plugins/loader.py", line 791, in _load_module_source
    spec.loader.exec_module(module)
  File "<frozen importlib._bootstrap_external>", line 843, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/usr/lib/python3.9/site-packages/ansible_runner/display_callback/callback/awx_display.py", line 316, in <module>
    class CallbackModule(DefaultCallbackModule):
TypeError: type 'NoneType' is not an acceptable base type
```

Note, I don't recall seeing the note about redirect before

Reply:
Commented it out for now

Reply:
Yeah, I don't think we strictly need that.
We get our reporting via the Foreman callback.